### ISSUE TITLE
Notes not appearing in InstantILL Open URLs

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#1647]
This test is failing: https://app.ghostinspector.com/tests/5f22f58c5e12c014aea2d989 because notes DONT appear to be included in the open url generated to submit the ILL. 

This is showing up on dev, but it likely an issues on live now too. Can you look into why we've lost notes?

**Comment by markmacgillivray** [comment]
Your description is a bit unclear, and I can't tell from the test you link to how it relates to what you say above, but based on the issue title I have assumed you mean you DO want notes to appear, so I have changed the embed behaviour to include notes on the open URL. I have tested it manually and checked that notes do appear. However the test you link to still fails, because it fails earlier than that when it tries to find a section of the page that does not appear to exist, and I'm not sure why you have it configured that way. It was already showing as failed at that point before I made any changes, so it's not clear how that relates to the notes issue at all. The title content your test seems to want to find does exist on the page, but just not at the place you configured it to look for, so is that just an error in the test? Or are you wanting me to also move the title content to a different location on the result page?

Have a look at the page itself now, and let me know if it is doing what you want, and adjust the test to what you want it to test and see if it passes. If so we can close this.
